### ISSUE TITLE
Match Tugboat MariaDB version with BRD RDS version

### ISSUE THREAD

**Issue by indytechcook** [department-of-veterans-affairs/va.gov-cms#4240]
The version of MariaDB was upgrade to 10.5 as part of #4226.  Tugboat and Lando versions should match BRD.

AC

- [x] Tugboat MariaDB is using version 10.5 - https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4303
- [x] Verify which command will propagate new database engine on Tugboat (This will be a rebuild, refresh will work too but will take longer, and this is only after the base image is updated after merge)
- [x] Change management/communication plan exists (if needed) to Demo users and engineers (none needed)

**Comment by ElijahLynn** [comment]
Tugboat has a MariaDB image: https://docs.tugboat.qa/reference/tugboat-images/#mysqlmariadbpercona

**Comment by ElijahLynn** [comment]
Lando is a change like this, although cannot specify the engine version it appears:
https://docs.lando.dev/config/drupal8.html#configuration (default is mysql)
```
recipe: drupal8
config:
  database: mariadb
```

**Comment by ElijahLynn** [comment]
Made PR for Lando to support 10.5. Even though it is on Bitnami, it wasn't in the allowlist for Lando. 

https://github.com/lando/lando/pull/2856

**Comment by ElijahLynn** [comment]
1. Tugboat PR upgrade here > https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4303. 
1. Lando PR upgrade here > https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4304 (blocked on upstream PR)

**Comment by ElijahLynn** [comment]
Breaking this out into its own story since it is dependent on an upstream PR:

> Lando MariaDB is using version 10.5 (blocked on upstream PR to Lando) - https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4304
